packs/menace-under-otari-bestiary/zombie-shambler-bb.json

```diff
@@ -143,7 +178,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>When the zombie hits with its fist Strike, it can use this action to automatically give the target the @UUID[Compendium.pf2e.conditionitems.Item.Grabbed] condition until the end of its next turn.</p>"
+                    "value": "<p>When the zombie succeeds at its fist Strike, it can use this action immediately after to attempt to @UUID[Compendium.pf2e.actionspf2e.Item.Grapple] the target . This Grapple doesn't apply or count toward the zombie's multiple attack penalty.</p>"
```

Comment:
```suggestion
                    "value": "<p>When the zombie succeeds at its fist Strike, it can use this action immediately after to attempt to @UUID[Compendium.pf2e.actionspf2e.Item.Grapple] the target. This Grapple doesn't apply or count toward the zombie's multiple attack penalty.</p>"
```